linera-views/src/backends/rocks_db.rs

```diff
@@ -114,205 +307,125 @@ impl ReadableKeyValueStore for RocksDbStoreInternal {
 
     async fn read_value_bytes(&self, key: &[u8]) -> Result<Option<Vec<u8>>, RocksDbStoreError> {
         ensure!(key.len() <= MAX_KEY_SIZE, RocksDbStoreError::KeyTooLong);
-        let client = self.clone();
-        let mut full_key = self.root_key.to_vec();
+        let db = self.executor.db.clone();
+        let mut full_key = self.executor.root_key.to_vec();
         full_key.extend(key);
-        Ok(tokio::task::spawn_blocking(move || client.db.get(&full_key)).await??)
+        Ok(match self.spawn_mode {
+            RocksDbSpawnMode::SpawnBlocking => {
```

Comment:
Can we have a method `RocksDbSpawnMode::spawn`?

Reply:
Possibly with `#[inline]` if we don't trust the compiler.

Reply:
I will try again. That did not seem worth it. We need 4 classes: the function, the input the underlying class (whether Arc<DB> (thus encapsulated), of RocksDbExecutor) considered and the output.

Reply:
Thank you. I could find some reasonable solution.